pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -20,60 +20,83 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"net/http"
 	"net/url"
 	"path"
-	"reflect"
 	"strings"
 	"time"
 
 	"github.com/kcp-dev/apimachinery/pkg/logicalcluster"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	metav1beta1 "k8s.io/apimachinery/pkg/apis/meta/v1beta1"
-	"k8s.io/cli-runtime/pkg/printers"
-	"k8s.io/client-go/rest"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/client-go/tools/clientcmd"
-	"k8s.io/client-go/tools/clientcmd/api"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 
-	virtualcommandoptions "github.com/kcp-dev/kcp/cmd/virtual-workspaces/options"
 	tenancyv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
 	tenancyv1beta1 "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1beta1"
 	tenancyclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
-	"github.com/kcp-dev/kcp/pkg/virtual/workspaces/registry"
 )
 
 const (
-	kcpPreviousWorkspaceContextKey         string = "workspace.kcp.dev/previous"
-	kcpCurrentWorkspaceContextKey          string = "workspace.kcp.dev/current"
-	kcpOverridesContextAuthInfoKey         string = "workspace.kcp.dev/overridden-auth"
-	kcpVirtualWorkspaceInternalContextName string = "workspace.kcp.dev/workspace-directory"
+	kcpPreviousWorkspaceContextKey string = "workspace.kcp.dev/previous"
+	kcpCurrentWorkspaceContextKey  string = "workspace"
```

Comment:
Curious: why did you rename from `workspace.kcp.dev/current` to `workspace`?

Reply:
`workspace.kcp.dev/current` appeared to be very cryptic, and one might use the workspace context manually. No strong opinion.

Related: https://github.com/kcp-dev/kcp/pull/779

Reply:
@davidfestal @ncdc opinions?

Reply:
I think I prefer the longer form, but I don't know we need to do too much bikeshedding since #779 should make this more user-driven?

Reply:
switched back to the longer form.